_opensearch/install/compatibility.md

```diff
@@ -17,3 +17,5 @@ nav_order: 2
 OpenSearch version | Compatible Java versions | Recommended operating systems
 :--- | :--- | :---
 1.x | 8, 11, 15 | Red Hat Enterprise Linux 7, 8; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04
+
+If you plan on running applications besides OpenSearch with different Java requirements, set the `OPENSEARCH_JAVA_HOME` environment variable to the location of the Java installation to be used by OpenSearch. The `OPENSEARCH_JAVA_HOME` supersedes the `JAVA_HOME` environment variable.
```

Comment:
Can we get rid of "The" in that last sentence? So, "`OPENSEARCH_JAVA_HOME` supersedes the `JAVA_HOME` environment variable."

Reply:
Hm ... I am not an expert but it looks grammatically correct (to me): every single occurrence of the environment variable it this document has `the` article.

Reply:
I agree with @keithhc2. It would be grammatically correct if we had "...environment variable" after the name of the variable. Since the second sentence just includes the name of the variable, there's no need for "the" article.

Reply:
Done, thanks!